### PR TITLE
feat: Update model evaluation notebook with sliced metrics

### PR DESCRIPTION
**Pull request by inardini**
<br>
This PR updates the automl_tabular_classification_model_evaluation  notebook to include sliced metrics. 
<br><br><br>

- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb [hunk 1]
```diff
@@ -8,7 +8,7 @@
       },
```
Comment (KevinBNaughton) [inline notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:8]: <pre class="ql-syntax" spellcheck="false">Line #143.        # sliced_classification_metrics = sliced_evaluation_metrics[0]["metrics"][
</pre><p>I believe to remove commented code. Multiple in this code block.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2038/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Comment (KevinBNaughton) [inline notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:8]: <pre class="ql-syntax" spellcheck="false">Line #25.        from google_cloud_pipeline_components.experimental.evaluation import (
</pre><p>Can you change imports to this?</p><p>from google_cloud_pipeline_components.v1.model_evaluation import ModelEvaluationClassificationOp</p><p>from google_cloud_pipeline_components.preview.evaluation import ModelEvaluationFeatureAttributionOp</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2038/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

Comment (KevinBNaughton) [inline notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:8]: <pre class="ql-syntax" spellcheck="false">Line #55.            model=get_model_task.outputs["model"],  # get_model_task.output,
</pre><p>Ditto</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2038/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

Comment (KevinBNaughton) [inline notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:8]: <pre class="ql-syntax" spellcheck="false">Line #98.            model=get_model_task.outputs["model"],  # get_model_task.output,
</pre><p>Remove commented code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2038/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

Comment (inardini) [reply notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:8]: done

Comment (inardini) [reply notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:8]: done

Comment (inardini) [reply notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:8]: done

Comment (inardini) [reply notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:8]: done

**Pull request closed without merging**